report.md

```diff
@@ -111,5 +118,24 @@ Notice that this data has 2-anonymity with respect to the attributes `Age`, `Gen
 
 ## Subsystem 4 (Secure Transfer)
 
+### Overview
+
+The other team will be given a special account (which is a type of user, such as `therapist` or `patient`) that can only perform one action: Upload their database to our database. 
+
+### Interface
+
+Upon logging into the special account, the other team will be redirected to a page whereby they can upload their database, image and video files separately. Our current database design is such that the image and video files are stored in a separate file system. The database stores the image and video file names. When a record is retrieved from the database, the server will also retrieve the corresponding image and video files based on their names.

```

Comment:
retrieve the corresponding image and video files based on a unique link to our file directory?

Reply:
Just to make sure we're on the same page:

The database will store the file names. So, whenever we retrieve a record from the database, it sees the file names and then retrieves the files from the file server accordingly?

Reply:
ohh ok. yup you are right, can check with adeeb to be sure!

Reply:
@chilipadiboy